source/craftercms_sphinx_theme/layout.html

```diff
@@ -91,6 +91,8 @@
 
   {% block head %} {% endblock %}
 
+  <link rel="stylesheet" href="https://unpkg.com/native-toast@2.0.0/dist/native-toast.css">
```

Comment:
You've imported it via package.json. You can probably import it on the theme stylesheet so it goes bundled in the theme's css file